iothub_client/samples/iothub_client_sample_module_sender/iothub_client_sample_module_sender.c

```diff
@@ -23,13 +23,13 @@
 
 /*String containing Hostname, Device Id & Device Key, ModuleID, and GatewayHostName in the format:                          */
 /*  "HostName=<host_name>;DeviceId=<device_id>;SharedAccessKey=<device_key>;ModuleId=<Module_Id>;GatewayHostName=127.0.0.1" */
-static const char* connectionString = "[device connection string]";
+static char* connectionString = "[device connection string]";
```

Comment:
Please leave this as a 'const char*'.  This means the value cannot be changed, but the pointer can, which is what we want.

Reply:
Sure.